src/ts-transformer/utils/template-literal-to-css.tsx

```diff
@@ -20,17 +22,22 @@ export const templateLiteralToCss = (
       throw new Error('variable doesnt exist in scope');
     }
 
-    if (!ts.isStringLiteral(value.initializer)) {
+    if (ts.isObjectLiteralExpression(value.initializer)) {
+      const processed = objectLiteralToCssString(value.initializer, scopedVariables, context);
+      css = processed.css;
+      cssVariables = processed.cssVariables;
```

Comment:
since we could have multiple of these i think this should be:

```suggestion
      cssVariables = cssVariables.concat(processed.cssVariables);
```